bindings/python/build_wheel.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e -x
+
+cd bindings/python
+
+# Install cargo
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source $HOME/.cargo/env
+sudo yum install openssl-devel.x86_64 -y
+cargo install cargo-c
+
+# Install unicorn globally
+rm -rf unicorn
+git clone -b dev https://github.com/unicorn-engine/unicorn unicorn
+cd unicorn
+mkdir build_dylib
```

Comment:
```suggestion
mkdir build
```
We shouldn't use `dylib` on Linux